### PR TITLE
introduce Util.sample_uniq/2

### PR DESCRIPTION
**Pull request by fenollp**
Generates some number of elements that are all unique

I've added:

- ~[ ] USAGE.md docs if applicable~
- [x] CHANGELOG.md

### CODE REVIEW

File: lib/faker/util.ex [hunk 1]
```diff
@@ -22,6 +22,28 @@ defmodule Faker.Util do
     Enum.at(list, Faker.random_between(0, Enum.count(list) - 1))
   end
 
+  @doc """
+  Generate N unique elements
+
+  ## Examples
+
+      iex> Faker.Util.sample_unique(2, &Faker.Internet.email/0)
+      ["conor2058@schiller.com", "elizabeth2056@rolfson.net"]
```
Comment (vbrazo) [inline lib/faker/util.ex]: We're using 4 examples for doctests. Check other modules out. Thanks.

Comment (fenollp) [reply lib/faker/util.ex]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 * `Faker.Color.En`, `Faker.Color.Es` and `Faker.Color.PtBr` [[@vbrazo][]]
 * `Faker.Internet` added Spanish and Brazilian Portuguese translations [[@vbrazo][]]
 * `Faker.Address.geohash/0` [[@anthonator][]] and [[@feyl][]]
+* `Faker.Util.sample_uniq/2` [[@fenollp][]]
```
Comment (igas) [inline CHANGELOG.md]: You also need to add a link at the bottom of .md

**Comment by igas** [comment]
Hi @fenollp,

I'm currently out of town. I'll be back in August and will take a look.

Thank you for your contribution.

**Comment by fenollp** [comment]
Hi there, any update on this?

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**